actions/drain_node_handler.go

```diff
@@ -195,7 +210,7 @@ func (h *drainNodeHandler) waitNodePodsTerminated(ctx context.Context, node *v1.
 
 // evictPod from the k8s node. Error handling is based on eviction api documentation:
 // https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#the-eviction-api
-func (h *drainNodeHandler) evictPod(ctx context.Context, pod v1.Pod) error {
+func (h *drainNodeHandler) evictPod(ctx context.Context, options metav1.DeleteOptions, pod v1.Pod) error {
```

Comment:
evict doesn't not need `DeleteOptions` so I would suggest keeping method signature as it was
and while deleting pods do:

```go

fn := func(ctx context.Context, pod v1.Pod) error {
	return h.deletePod(ctx, options, pod)
}

if err := h.sendPodsRequests(ctx, pods, fn); err != nil {
	return fmt.Errorf("sending delete pods requests: %w", err)
}


```